src/handlers/interventions-processor-handler.ts

```diff
@@ -104,8 +104,16 @@ async function processSQSRecord(record: SQSRecord) {
     recordBody,
     statusResult.interventionName,
   );
-  logger.debug('processed requested event, sending update request to dynamo db');
+  logger.debug('Updating user status', { userId, partialCommandInput });
   await service.updateUserStatus(userId, partialCommandInput);
+  publishTimeToResolveMetrics(
+    currentAccountState,
+    statusResult.stateResult,
+    itemFromDB?.appliedAt ? itemFromDB.appliedAt : currentTimestamp.milliseconds,
```

Comment:
linting is complaining about the ternary here.. I think `itemFromDB?.appliedAt || currentTimestamp.milliseconds` is equivalent and should satisfy the liniting